lib/vagrant-parallels/action/import.rb

```diff
@@ -20,32 +22,54 @@ def call(env)
             env[:machine].provider.driver.disable_password_restrictions(acts)
           end
 
-          # Register template to be able to clone it further
-          register_template(template_path.to_s)
+          # Linked clones are supported only for PD 11 and higher
+          if env[:machine].provider_config.linked_clone \
+            && env[:machine].provider.pd_version_satisfies?('>= 11')
+            # Linked clone creation should not be concurrent [GH-206]
+            @@lock.synchronize do
+              lock_key = Digest::MD5.hexdigest("#{env[:clone_id]}-linked-clone")
+              env[:machine].env.lock(lock_key, retry: true) do
+                clone_linked(env)
+              end
+            end
+          else
+            clone_full(env)
+          end
 
-          # Get template name. It might be changed during registration if name
-          # collision has been occurred
-          tpl_name = template_name(template_path)
+          # If we got interrupted, then the import could have been
+          # interrupted and its not a big deal. Just return out.
+          return if env[:interrupted]
 
-          # Import VM, e.q. clone it from registered template
-          import(env, tpl_name)
+          # Flag as erroneous and return if import failed
+          raise Errors::VMCloneFailure if !env[:machine].id
 
-          # Hide template since we dont need it anymore
-          unregister_template(tpl_name)
+          if env[:machine].provider_config.regen_src_uuid \
+            && env[:machine].provider.pd_version_satisfies?('< 11')
+            @logger.info('Regenerate SourceVmUuid by editing config.pvs file')
+            env[:machine].provider.driver.regenerate_src_uuid
+          end
 
-          # Flag as erroneous and return if import failed
-          raise Errors::VMImportFailure if !@machine.id
+          # Remove 'Icon\r' file from VM home (bug in PD 11.0.0)
```

Comment:
Yes, there is some control character in the filename. Most likely it is a carriage return. <facepalm.jpg>
It was fixed in PD 11.0.1